src/utils.ts

```diff
@@ -68,20 +69,20 @@ export function createSignInputsFromImportableEd25519Key(key: TImportableEd25519
     throw new Error('No verification method type provided')
 }
 
-export function createKeyPairRaw(): KeyPair {
-    return generateKeyPair()
+export function createKeyPairRaw(seed?: string): KeyPair {
+    return seed ? generateKeyPairFromSeed(Buffer.from(seed)) : generateKeyPair()
```

Comment:
Can we switch to `uint8arrays` for byte array operations instead?
Generally, `buffer` does not play well with browser environments and has memory management flaws, hence it's deprecated and not suggested for production usage.